ci-operator/step-registry/hypershift/kubevirt/health-check/hypershift-kubevirt-health-check-commands.sh

```
@@ -14,11 +14,16 @@ fi
 CLUSTER_NAMESPACE=${CLUSTER_NAMESPACE_PREFIX}-${CLUSTER_NAME}
 
 echo "Waiting for nested cluster's node count to reach the desired replicas count in the NodePool"
+WAIT_TIMEOUT=$(($(date +%s) + 1800)) # 30 minutes
 until \
-  [[ $(oc get nodepool ${CLUSTER_NAME} -n ${CLUSTER_NAMESPACE_PREFIX} -o jsonpath='{.spec.replicas}') \
-    == $(oc --kubeconfig=${SHARED_DIR}/nested_kubeconfig get nodes --no-headers | wc -l) ]]; do
+  [[ $(oc get nodepool ${CLUSTER_NAME} -n ${CLUSTER_NAMESPACE_PREFIX} -o jsonpath='{.spec.replicas}' 2>/dev/null || echo "") \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)